### DISCUSSION TITLE
how do I use bg-base-200 bg-base-300 with a dark theme

### DISCUSSION THREAD

**Discussion by wighawag** [saadeghi/daisyui#1615] [General]
how do I use bg-base-200 bg-base-300 with a dark theme

In a dark them where the backgroudn is black bg-base-200 and bg-base-300 do not have visible effect or am I doing something wrong ?

**Comment by saadeghi** [comment]
The idea is `base-300` being the darkest one and `base-100` to be a little lighter (not pure black).  
This way we can always have a darker shade if we use `base-200` or `base-300` either if it's a light theme or a dark theme.

This is the suggested color shade but of course you can customize them. For example if you want to use `bg-100` for the background of the page and you want it to be `#000` you can then set lighter colors for `base-200` and `base-300`.

There are built-in themes in daisyUI they ignore that different. For example in `black` them, it's all black and it's just a example if using less vs color shades but you might want to customize the themes and colors the way you want.

Let me know if you have any questions
